runtime/j9vm/java11vmi.c

```diff
@@ -1812,7 +1812,24 @@ JVM_GetNestMembers(JNIEnv *env, jclass clz)
 JNIEXPORT jboolean JNICALL
 JVM_AreNestMates(JNIEnv *env, jclass clzOne, jclass clzTwo)
 {
-	assert(!"JVM_AreNestMates unimplemented");
+	j9object_t clazzOne = NULL;
+	j9object_t clazzTwo = NULL;
```

Comment:
These two variable defines could be moved into next `if` block and close to their usages.